### PR TITLE
feat: render asset path

### PR DESCRIPTION
**Pull request by cfitzw**
a corrupt asset may exist from time-to-time and this fix aims to make it easier to locate that asset by showing the path regardless

### CODE REVIEW

**Comment by alextran1502** [comment]
Hello, can you capture screenshots of before/after to show the differences?

File: web/src/lib/components/asset-viewer/detail-panel.svelte [hunk 1]
```diff
@@ -345,7 +345,7 @@
       </Portal>
     {/if}
 
-    {#if asset.exifInfo?.fileSizeInByte}
+    {#if true}
```
Comment (danieldietzler) [inline web/src/lib/components/asset-viewer/detail-panel.svelte:348]: Can we... Not? 😅

File: web/src/lib/components/asset-viewer/detail-panel.svelte [hunk 2]
```diff
@@ -362,18 +362,20 @@
               />
             {/if}
           </p>
-          <div class="flex gap-2 text-sm">
-            {#if asset.exifInfo.exifImageHeight && asset.exifInfo.exifImageWidth}
-              {#if getMegapixel(asset.exifInfo.exifImageHeight, asset.exifInfo.exifImageWidth)}
-                <p>
-                  {getMegapixel(asset.exifInfo.exifImageHeight, asset.exifInfo.exifImageWidth)} MP
-                </p>
+          {#if asset.exifInfo?.fileSizeInByte && asset.exifInfo?.exifImageHeight && asset.exifInfo?.exifImageWidth}
```
Comment (danieldietzler) [inline web/src/lib/components/asset-viewer/detail-panel.svelte:365]: We haven't had a condition around this block before, why did you add it now?

File: web/src/lib/components/asset-viewer/detail-panel.svelte (hunk 1 above)
Comment (cfitzw) [reply web/src/lib/components/asset-viewer/detail-panel.svelte]: Yes, we can... not ;).

File: web/src/lib/components/asset-viewer/detail-panel.svelte (hunk 2 above)
Comment (cfitzw) [reply web/src/lib/components/asset-viewer/detail-panel.svelte]: Improved this further in most recent commit. The conditions now better align now and is similar to other structures in this file (check to see if exif data exists, if it does, display the block and data).

I also moved the file path to show directly below the filename when info button is clicked. A little more intuitive to see it show right next to what you are clicking on (as well as keep that code together).

**Review by danieldietzler — APPROVED** [review]

**Review by alextran1502 — APPROVED** [review]
